packages/truffle-core/lib/commands/debug.js

```diff
@@ -402,6 +423,21 @@ var command = {
             }
           }
 
+          function watchExpressionAnalytics(raw) {
+            if (raw.includes("!<")) {
+              //don't send analytics for watch expressions involving selectors
+              return;
+            }
+            let expression = raw.trim();
+            //legal Solidity identifiers (= legal JS identifiers)
+            let identifierRegex = /^[a-zA-Z_$][a-zA-Z_$0-9]*$/;
+            let isVariable = expression.match(identifierRegex) !== null;
+            analytics.send({
+              command: "watch expression",
```

Comment:
Is there a way to scope this within `debug`? If it's not straightforward then it's not necessary, but just curious.

Reply:
I don't think there's any formal way, but I could add "`debug:`" or something to the beginning.  The impression I got from @fainashalts was that we don't have any current convention for this, but I guess this could be the start of one!

Reply:
Adding `debug:` seems like a valid solution to this issue. Would definitely make it clearer where this is coming from, on the analytics side of things. Another option would be to add the watch expression part to the args -- could make args "watch expression true" and "watch expression false" since its a string anyway on the analytics side. I'm ok with either option!

Reply:
Oh so like `command = "debug"`, `args = {type:"watch expression", isVariable}`.  That makes sense.

Reply:
Hm, @gnidan points out that actually `command = "debug"` is probably not a good idea, as right now `command = "debug"` always means that `debug` is being invoked.  So going with the colon way after all.